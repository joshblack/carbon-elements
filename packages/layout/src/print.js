/**
 * Copyright IBM Corp. 2018, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

/**
 * Print the given unit with its value. Useful for printing CSS values like
 * `100%`
 * @param {string} unit
 * @return {(value: string) => string}
 */
export function print(unit) {
  return value => {
    return `${value}${unit}`;
  };
}

/**
 * Printer for units that are unitless when they have a zero value. Useful for
 * things like `rem` values where zero is printed as `0`
 * @param {string} unit
 * @return {(value: string) => string}
 */
export function unitless(unit) {
  const _print = print(unit);
  return value => {
    if (value === 0) {
      return '0';
    }
    return _print(value);
  };
}
