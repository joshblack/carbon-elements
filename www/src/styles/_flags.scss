$feature-flags: (
  ui-shell: true,
);

$css--font-face: false;
$css--helpers: true;
$css--body: false;
$css--use-layer: true;
$css--reset: true;
$css--typography: false;
$css--plex: false;
