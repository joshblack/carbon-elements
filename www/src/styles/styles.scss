@import 'flags';
@import '~carbon-components/scss/components/ui-shell/ui-shell';
@import '@carbon/grid/scss/grid';
@import '@carbon/type/scss/type';
@import 'components/components';

html,
body,
#___gatsby,
#___gatsby > [role='group'] {
  width: 100%;
  height: 100%;
}

body {
  overflow: hidden;
}

#___gatsby > [role='group'] > * {
  overflow-y: auto;
}
