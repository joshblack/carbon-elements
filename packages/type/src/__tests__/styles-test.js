/**
 * @jest-environment node
 */

import * as tokens from '../styles';
import { print } from '../print';

describe('styles', () => {
  test.each(Object.keys(tokens))('%s should be printable', key => {
    const token = tokens[key];
    expect(token).toMatchSnapshot();
    expect(print(token)).toMatchSnapshot();
  });
});
