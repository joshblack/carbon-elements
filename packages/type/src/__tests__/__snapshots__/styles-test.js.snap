// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles bodyLong01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.875rem",
  "fontWeight": 400,
  "letterSpacing": "0.16px",
  "lineHeight": "1.25rem",
}
`;

exports[`styles bodyLong01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.875rem;
font-weight: 400;
line-height: 1.25rem;
letter-spacing: 0.16px;"
`;

exports[`styles bodyLong02 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "1rem",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "1.5rem",
}
`;

exports[`styles bodyLong02 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 1rem;
font-weight: 400;
line-height: 1.5rem;
letter-spacing: 0;"
`;

exports[`styles bodyShort01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.875rem",
  "fontWeight": 400,
  "letterSpacing": "0.16px",
  "lineHeight": "1.125rem",
}
`;

exports[`styles bodyShort01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.875rem;
font-weight: 400;
line-height: 1.125rem;
letter-spacing: 0.16px;"
`;

exports[`styles bodyShort02 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "1rem",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "1.375rem",
}
`;

exports[`styles bodyShort02 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 1rem;
font-weight: 400;
line-height: 1.375rem;
letter-spacing: 0;"
`;

exports[`styles caption01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.75rem",
  "fontWeight": 400,
  "letterSpacing": "0.32px",
  "lineHeight": "1rem",
}
`;

exports[`styles caption01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.75rem;
font-weight: 400;
line-height: 1rem;
letter-spacing: 0.32px;"
`;

exports[`styles code01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace",
  "fontSize": "0.75rem",
  "fontWeight": 400,
  "letterSpacing": "0.32px",
  "lineHeight": "1rem",
}
`;

exports[`styles code01 should be printable 2`] = `
"font-family: 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace;
font-size: 0.75rem;
font-weight: 400;
line-height: 1rem;
letter-spacing: 0.32px;"
`;

exports[`styles code02 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace",
  "fontSize": "0.875rem",
  "fontWeight": 400,
  "letterSpacing": "0.32px",
  "lineHeight": "1.25rem",
}
`;

exports[`styles code02 should be printable 2`] = `
"font-family: 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace;
font-size: 0.875rem;
font-weight: 400;
line-height: 1.25rem;
letter-spacing: 0.32px;"
`;

exports[`styles display01 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontSize": "calc(2.625rem + 0.75 * ((100vw - 42rem) / 24))",
  },
  "@media (min-width: 66rem)": Object {
    "fontSize": "calc(3.375rem + 0.375 * ((100vw - 66rem) / 16))",
  },
  "@media (min-width: 82rem)": Object {
    "fontSize": "calc(3.75rem + 1 * ((100vw - 82rem) / 17))",
    "lineHeight": "117%",
  },
  "@media (min-width: 99rem)": Object {
    "fontSize": "4.75rem",
    "lineHeight": "113%",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(2.625rem + 0 * ((100vw - 20rem) / 22))",
  "fontWeight": 300,
  "letterSpacing": 0,
  "lineHeight": "119%",
}
`;

exports[`styles display01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(2.625rem + 0 * ((100vw - 20rem) / 22));
font-weight: 300;
line-height: 119%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles display02 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontSize": "calc(2.625rem + 0.75 * ((100vw - 42rem) / 24))",
  },
  "@media (min-width: 66rem)": Object {
    "fontSize": "calc(3.375rem + 0.375 * ((100vw - 66rem) / 16))",
  },
  "@media (min-width: 82rem)": Object {
    "fontSize": "calc(3.75rem + 1 * ((100vw - 82rem) / 17))",
    "lineHeight": "116%",
  },
  "@media (min-width: 99rem)": Object {
    "fontSize": "4.75rem",
    "lineHeight": "113%",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(2.625rem + 0 * ((100vw - 20rem) / 22))",
  "fontWeight": 600,
  "letterSpacing": 0,
  "lineHeight": "119%",
}
`;

exports[`styles display02 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(2.625rem + 0 * ((100vw - 20rem) / 22));
font-weight: 600;
line-height: 119%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles display03 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontSize": "calc(4.25rem + 1.5 * ((100vw - 42rem) / 24))",
    "lineHeight": "115%",
  },
  "@media (min-width: 66rem)": Object {
    "fontSize": "calc(5.75rem + 1.875 * ((100vw - 66rem) / 16))",
    "letterSpacing": "-0.64px",
    "lineHeight": "111%",
  },
  "@media (min-width: 82rem)": Object {
    "fontSize": "calc(7.625rem + 2.125 * ((100vw - 82rem) / 17))",
    "lineHeight": "107%",
  },
  "@media (min-width: 99rem)": Object {
    "fontSize": "9.75rem",
    "letterSpacing": "-0.96px",
    "lineHeight": "105%",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(2.625rem + 1.625 * ((100vw - 20rem) / 22))",
  "fontWeight": 300,
  "letterSpacing": 0,
  "lineHeight": "119%",
}
`;

exports[`styles display03 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(2.625rem + 1.625 * ((100vw - 20rem) / 22));
font-weight: 300;
line-height: 119%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles display04 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontSize": "calc(4.25rem + 1.5 * ((100vw - 42rem) / 24))",
    "lineHeight": "115%",
  },
  "@media (min-width: 66rem)": Object {
    "fontSize": "calc(5.75rem + 1.875 * ((100vw - 66rem) / 16))",
    "letterSpacing": "-0.64px",
    "lineHeight": "111%",
  },
  "@media (min-width: 82rem)": Object {
    "fontSize": "calc(7.625rem + 2.125 * ((100vw - 82rem) / 17))",
    "letterSpacing": "-0.64px",
    "lineHeight": "107%",
  },
  "@media (min-width: 99rem)": Object {
    "fontSize": "9.75rem",
    "letterSpacing": "-0.96px",
    "lineHeight": "105%",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(2.625rem + 1.625 * ((100vw - 20rem) / 22))",
  "fontWeight": 600,
  "letterSpacing": 0,
  "lineHeight": "119%",
}
`;

exports[`styles display04 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(2.625rem + 1.625 * ((100vw - 20rem) / 22));
font-weight: 600;
line-height: 119%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles expressiveHeading04 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(1.25rem + 0.25 * ((100vw - 42rem) / 24))",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "1.625rem",
  },
  "@media (min-width: 66rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(1.5rem + 0.25 * ((100vw - 66rem) / 16))",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "125%",
  },
  "@media (min-width: 82rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(1.75rem + 0.25 * ((100vw - 82rem) / 17))",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "129%",
  },
  "@media (min-width: 99rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "2rem",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "125%",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(1.25rem + 0 * ((100vw - 20rem) / 22))",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "130%",
}
`;

exports[`styles expressiveHeading04 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(1.25rem + 0 * ((100vw - 20rem) / 22));
font-weight: 400;
line-height: 130%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles expressiveHeading05 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(2.25rem + 0.375 * ((100vw - 42rem) / 24))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "122%",
  },
  "@media (min-width: 66rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(2.625rem + 0.375 * ((100vw - 66rem) / 16))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "119%",
  },
  "@media (min-width: 82rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "calc(3rem + 0.75 * ((100vw - 82rem) / 17))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "117%",
  },
  "@media (min-width: 99rem)": Object {
    "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
    "fontSize": "3.75rem",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "4.375rem",
  },
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "calc(1.75rem + 0.5 * ((100vw - 20rem) / 22))",
  "fontWeight": 300,
  "letterSpacing": 0,
  "lineHeight": "129%",
}
`;

exports[`styles expressiveHeading05 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: calc(1.75rem + 0.5 * ((100vw - 20rem) / 22));
font-weight: 300;
line-height: 129%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles heading01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.875rem",
  "fontWeight": 600,
  "letterSpacing": "0.16px",
  "lineHeight": "1.125rem",
}
`;

exports[`styles heading01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.875rem;
font-weight: 600;
line-height: 1.125rem;
letter-spacing: 0.16px;"
`;

exports[`styles heading02 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "1rem",
  "fontWeight": 600,
  "letterSpacing": 0,
  "lineHeight": "1.375rem",
}
`;

exports[`styles heading02 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 1rem;
font-weight: 600;
line-height: 1.375rem;
letter-spacing: 0;"
`;

exports[`styles heading03 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "1.25rem",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "1.625rem",
}
`;

exports[`styles heading03 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 1.25rem;
font-weight: 400;
line-height: 1.625rem;
letter-spacing: 0;"
`;

exports[`styles helperText01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.75rem",
  "fontStyle": "italic",
  "letterSpacing": "0.32px",
  "lineHeight": "1rem",
}
`;

exports[`styles helperText01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.75rem;
font-style: italic;
line-height: 1rem;
letter-spacing: 0.32px;"
`;

exports[`styles label01 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "0.75rem",
  "fontWeight": 400,
  "letterSpacing": "0.32px",
  "lineHeight": "1rem",
}
`;

exports[`styles label01 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 0.75rem;
font-weight: 400;
line-height: 1rem;
letter-spacing: 0.32px;"
`;

exports[`styles productiveHeading04 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "1.5rem",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "1.875rem",
}
`;

exports[`styles productiveHeading04 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 1.5rem;
font-weight: 400;
line-height: 1.875rem;
letter-spacing: 0;"
`;

exports[`styles productiveHeading05 should be printable 1`] = `
Object {
  "fontFamily": "'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif",
  "fontSize": "2rem",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "2.5rem",
}
`;

exports[`styles productiveHeading05 should be printable 2`] = `
"font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
font-size: 2rem;
font-weight: 400;
line-height: 2.5rem;
letter-spacing: 0;"
`;

exports[`styles quotation01 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(1.25rem + 0.25 * ((100vw - 42rem) / 24))",
    "fontWeight": 400,
    "letterSpacing": 0,
  },
  "@media (min-width: 66rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(1.5rem + 0.25 * ((100vw - 66rem) / 16))",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "125%",
  },
  "@media (min-width: 82rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(1.75rem + 0.25 * ((100vw - 82rem) / 17))",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "129%",
  },
  "@media (min-width: 99rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "2rem",
    "fontWeight": 400,
    "letterSpacing": 0,
    "lineHeight": "125%",
  },
  "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
  "fontSize": "calc(1.25rem + 0 * ((100vw - 20rem) / 22))",
  "fontWeight": 400,
  "letterSpacing": 0,
  "lineHeight": "130%",
}
`;

exports[`styles quotation01 should be printable 2`] = `
"font-family: 'IBM Plex Serif', 'Georgia', Times, serif;
font-size: calc(1.25rem + 0 * ((100vw - 20rem) / 22));
font-weight: 400;
line-height: 130%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;

exports[`styles quotation02 should be printable 1`] = `
Object {
  "@media (min-width: 42rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(2.25rem + 0.375 * ((100vw - 42rem) / 24))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "122%",
  },
  "@media (min-width: 66rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(2.625rem + 0.375 * ((100vw - 66rem) / 16))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "119%",
  },
  "@media (min-width: 82rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "calc(3rem + 0.75 * ((100vw - 82rem) / 17))",
    "fontWeight": 300,
    "letterSpacing": 0,
    "lineHeight": "117%",
  },
  "@media (min-width: 99rem)": Object {
    "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
    "fontSize": "3.75rem",
    "fontWeight": 300,
    "letterSpacing": 0,
  },
  "fontFamily": "'IBM Plex Serif', 'Georgia', Times, serif",
  "fontSize": "calc(1.75rem + 0.5 * ((100vw - 20rem) / 22))",
  "fontWeight": 300,
  "letterSpacing": 0,
  "lineHeight": "129%",
}
`;

exports[`styles quotation02 should be printable 2`] = `
"font-family: 'IBM Plex Serif', 'Georgia', Times, serif;
font-size: calc(1.75rem + 0.5 * ((100vw - 20rem) / 22));
font-weight: 300;
line-height: 129%;
letter-spacing: 0;
-@media- -(min--width-:- -4-2rem-): [object Object];
-@media- -(min--width-:- -6-6rem-): [object Object];
-@media- -(min--width-:- -8-2rem-): [object Object];
-@media- -(min--width-:- -9-9rem-): [object Object];"
`;
